Sources/SnowManLogic/ViewLogic/MockUI.swift

```diff
@@ -0,0 +1,13 @@
+public class MockUI: Receivable {
+	public var getUserInputReturn: String!
```

Comment:
I know we came up with this name quickly when we were going over this together...but, this name could definitely be improved